client/viame-web-common/components/Viewer.vue

```diff
@@ -429,7 +429,7 @@ export default defineComponent({
       style="min-width: 700px;"
     >
       <sidebar
-        v-bind="{ newTrackSettings, typeSettings }"
+        v-bind="{ newTrackSettings, typeSettings, datasetId:id }"
```

Comment:
It's possible that `provides.State` would have been a better place to put this, especially considering everything in state belongs to exactly one dataset.  I'm not going to dwell on this, it's good like it is for now.

Reply:
My brain was broken at the moment.  I knew internally that this is something that would be in a global store in vuex terms and the provides stuff just slipped my mind for some reason.  I'll look into modifying it and providing it from there.